NEW_README.md

```diff
@@ -49,4 +49,115 @@ Follow these steps to get started with RUN:
 
 3. **Run the task and see task output**
       
-    <img alt="hello-gif" src="screenshots/hello.gif">
\ No newline at end of file
+    <img alt="hello-gif" src="screenshots/hello.gif">
+
+## Configuration
```

Comment:
review note: the changes in this commit are,
- add a new "configuration" section which introduces the task fields without going into detail about all of them; it hints a new "task management" section which would have full documentation about each field
- incorporate the existing "CLI Usage" and "User Interfaces" sections unchanged

Reply:
review:
a quick overview of configuration is a great idea; i stan.